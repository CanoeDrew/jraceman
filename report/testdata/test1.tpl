<html>
<body>
Sample report.
Top: {{.}}
</body>
</html>
