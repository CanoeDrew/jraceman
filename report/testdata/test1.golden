<html>
<body>
Sample report.
Top: &lt;topdata&gt;
</body>
</html>
